### PR TITLE
Updated episode 18 dialogues

### PR DESCRIPTION
**Pull request by Atemo**
* **Addressed Issue(s)**: N/A

* **Server Mode**: renewal

* **Description of Pull Request**: 

Updated episode 18 dialogues using iRO text.
Renamed `ep18_1_main` variable by `ep18_main`.

### CODE REVIEW

**Comment by monbazko** [comment]
Mejai daily quest seems bugged and not taking new quest after cooldown.
Mejai#E18_7 at line 12215
I think if in line 12257 and else at line 12259 is the culprit

**Comment by Atemo** [comment]
Capture d’écran (2004)

The quest works fine for me (after cooldown), please try again.

**Comment by monbazko** [comment]
> The quest works fine for me (after cooldown), please try again.

I did, still didn't work

`line 12257`
```
	if (checkquest(5937,PLAYTIME) != 2
		npctalk "Mejai: Hello. Let's do our best today too.", "Mejai#E18_7", bc_self;
	else {
		npctalk "Mejai: Come on! Today's work is done.", "Mejai#E18_7", bc_self;
		sleep2 1000;
		setpcblock PCBLOCK_NPC, false;
		npctalk "Mejai: Go home and be faithful to your family. It's not like this is the only thing you have to do.", "Mejai#E18_7", bc_self;
		end;
	}
	sleep2 1000;
	setpcblock PCBLOCK_NPC, false;
	switch( select( "I'm currently busy..", "Harvesting Half Flowers", "Delivering Half Flowers" ) ){
```

shouldn't it be == 2, cuz it reads that i didn't qualify the if so it always read the else

File: npc/re/quests/quests_18.txt [hunk 1]
```diff
@@ -0,0 +1,13096 @@
...
+	mes "Someone who can't understand a word! Someone stupid! Someone who makes you say things again! Someone who shows emotions in their work!";
+	next;
+	mes "[Redcell]";
+	mes "Someone who hides their mistakes and then can't undo them! Someone who can't take responsibility for their own actions!";
+	next;
+	mes "[Redcell]";
+	mes "It's just disgusting!";
+	next;
+	mes "[Redcell]";
+	mes "Why do so many people hate you like th
```
Comment (reunite-ro) [inline npc/re/quests/quests_18.txt:13024]: Since I create 1 issues regarding this, is it iRO selection for Coat is regarded as 'footwear' here ?

Comment (Atemo) [reply npc/re/quests/quests_18.txt:13024]: Yes it's a bad translation from iRO

**Pull request merged**